.github/workflows/blackduck-scan.yml

```diff
@@ -1,45 +1,61 @@
-name: BlackDuck-Scan-Workflow
+###
+##  Foundation-security BlackDuck workflow 
+#
+name: Foundation-Security/Black Duck Scan
 
 on:
-  # Triggers the workflow on push request events but only for the master branch
-  pull_request:
-    branches: [ master ]
-
-  # Allows you to run this workflow manually from the Actions tab
+  push:
+    tags:        
+      - '**'
   workflow_dispatch:
 
 jobs:
-  scan:
-    name: BlackDuck-scan
-    # The type of runner that the job will run on
-    runs-on: ubuntu-latest
-    if: github.event.pull_request.head.repo.full_name == 'EnterpriseDB/barman'
-
+  Blackduck-Scan:
+    runs-on: ubuntu-20.04
     steps:
-      - name: Checkout code
-        uses: actions/checkout@v2
-      - name: Set up JDK 8
-        uses: actions/setup-java@v2.1.0
-        with:
-          java-version: 8
-          distribution: 'zulu'
-      - name: Install python
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.7
-      - name: Install project
-        run: |
-          python -m pip install --upgrade pip setuptools wheel
-          python -m pip install -e .[cloud]
-          python -m pip install -e .[completion]
-          python -m pip install -e .[azure]
-      # -- Build with steps and requirements
-      - name: Synopsys Detect
-        uses: blackducksoftware/github-action@2.0.0
-        with:
-          args: '--blackduck.url="${{ secrets.BLACKDUCK_URL}}" --blackduck.api.token="${{ secrets.BLACKDUCK_API_TOKEN}}" --detect.project.name="Barman" --detect.project.version.name="CI/CD" --detect.risk.report.pdf=true --detect.excluded.detector.types="npm,cpan,nuget,git,maven,lerna" --detect.blackduck.signature.scanner.license.search'
+    - name: Checkout source repository
+      id: checkout-source
+      uses: actions/checkout@v3
+
+    - name: Checkout foundation repository
+      id: checkout-foundation
+      uses: actions/checkout@v3
+      with:
+          repository: EnterpriseDB/foundation-packaging
```

Comment:
Not sure foundation-packaging can be checkout from barman repository as it is a public repository and it does not inherit enterprise secrets.

Reply:
This is a very good point. I wonder if it would be more appropriate to add this scanning to https://github.com/EnterpriseDB/barman-packaging instead of the OSS repo?

Reply:
Alright, that is good to know about the secrets. In any case, Foundation Security changes need to be made to Sonarqube and Blackduck. Please make me aware of the desired repo, with write permissions.

Reply:
I can see that barman-packaging already uses foundation with its branch. No scanning is present in that repo though.